src/ReactiveUI.Events.XamForms/ReactiveUI.Events.XamForms.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>netstandard1.3</TargetFrameworks>
+    <PackageTargetFallback Condition="'$(TargetFramework)' == 'netstandard1.3'">
+      $(PackageTargetFallback);portable-win+net45+wp8+win81+wpa8
+    </PackageTargetFallback>
+    <AssemblyName>ReactiveUI.Events</AssemblyName>
+    <RootNamespace>ReactiveUI.Events</RootNamespace>
+    <Description>Provides Observable-based events API for Xamarin Forms UI controls/eventhandlers. The contents of this package is automatically generated, please target pull-requests to the code generator.</Description>
+    <PackageId>reactiveui-events-xamforms</PackageId>
+  </PropertyGroup>  
+
+  <ItemGroup>
+    <Compile Remove="*.cs" />
+    <None Include="*.cs" />
+    <Compile Include="../ReactiveUI.Events/SingleAwaitSubject.cs" />
+    <PackageReference Include="System.Reactive" Version="3.1.0" />
+  </ItemGroup>
+
+  <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard1.3' ">
```

Comment:
Don't know if it's worth just dropping the conditional here? We expect this to only build for netstandard 1.3, and only for XamForms don't we?